cypress/utils/constants.js

```diff
@@ -50,3 +50,32 @@ export const DETECTOR_STATE_INIT = 'Initializing';
 export const DETECTOR_STATE_RUNNING = 'Running';
 
 export const DETECTOR_STATE_FEATURE_REQUIRED = 'Feature required';
+
+/**
+ *****************************
+ IM PLUGIN CONSTANTS
```

Comment:
looks like this file will grow larger than expected. Probably you can create a separate file for ISM constants. 

cc @ohltyler

Reply:
I think it makes sense. 

> Agreed that this may grow too large and can lead to annoying merge conflicts. I think a good solution may be moving these sorts of files (`constants.js`, `helpers.js`, etc.) within the plugin directory (e.g., `plugins/index-management-dashboards-plugin/constants.js`.
> 
> @bowenlan-amzn @tianleh what do you think?

LGTM.

Reply:
Thanks, I will follow this way.